app/views/coronavirus_form/are_you_a_manufacturer.html.erb

```diff
@@ -23,7 +23,7 @@
     {
       value:  item[:label],
       label:  item[:label],
-      checked: session.fetch("are_you_a_manufacturer", []).include?(item[:label]),
+      checked: @form_responses.fetch(:$1, []).include?(item[:label]),
```

Comment:
Oooh I haven't used `:$1` like this before. Was this change it response to the linter?

Reply:
Can we be sure what `$1` is?

Reply:
This was a find and replace fail (which only failed in some of the views).  Have fixed.

Reply:
Wasn't this supposed to be replaced?

Reply:
Have actually committed the change this time